src/main/typescript/components/Header.tsx

```diff
@@ -79,17 +78,14 @@ const NavBar: FC = ({ children }) => (
     </div>
 )
 
-interface HeaderProps {
-    isLoggedIn: boolean
-    loginName: string
+const Header = () => {
+    const isLoggedIn = useSelector(state => state.authenticatedUser.isAuthenticated)
```

Comment:
I'm a bit worried about the distinction between logged-in and authenticated. We are not working with a session concept any more. It might cause the loops reported for #170 

Each time you need any service request to build/update a page you might discover there is no valid cookie any more. I guess the strategy should become something like: just try to build the page, see [EASY-2166](https://drivenbydata.atlassian.net/browse/EASY-2166) as soon as you run in a 403 response by the service. This redesign will need another pull request.